### PR TITLE
Actview

### PR DESCRIPTION
**Pull request by hidde-jan**
Extract ActView from ActorView to clean up code somewhat

### CODE REVIEW

File: compliance-by-design-demo/src/components/ActButton.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+import Button from "react-bootstrap/Button";
+
+export const SUBSIDY_TITLES = {
+  "<<indienen aanvraag tegemoetkoming in de schade geleden door de maatregelen ter bestrijding van de verdere verspreiding van COVID-19>>":
+    "Tegemoedkoming geleden schade ten gevolge van de Coronamaatregelen",
```
Comment (pimotte) [inline compliance-by-design-demo/src/components/ActButton.js:8]: Tegemoetkoming*

File: compliance-by-design-demo/src/components/KvkForm.js [hunk 2]
```diff
@@ -75,49 +79,201 @@ class KvkForm extends Component {
...
-          >
-            Confirm
-          </Button>
-        </div>
+        <Container>
+          <Form onSubmit={this.returnDerivedFacts.bind(this)}>
+            <h2>Controleer uw gegevens</h2>
+            <p>
+              We hebben de volgende gegevens voor uw bedrijf opgehaald.
+              Controlleer ze goed en pas aan wat niet klopt.
```
Comment (pimotte) [inline compliance-by-design-demo/src/components/KvkForm.js:156]: Controleer*

File: compliance-by-design-demo/src/model/tegemoetkoming-schade-covid19.flint.json.old [hunk 3]
```diff
@@ -0,0 +1,1426 @@
+{
```
Comment (pimotte) [inline compliance-by-design-demo/src/model/tegemoetkoming-schade-covid19.flint.json.old:1]: I guess we can remove this one?

**Comment by pimotte** [comment]
Nice work! Couple spelling errors and an extra file, but then this should be ready for merge.

It looks like this makes #6 obsolete, since this includes a newer model, right?

File: compliance-by-design-demo/src/model/tegemoetkoming-schade-covid19.flint.json.old (hunk 3 above)
Comment (hidde-jan) [reply compliance-by-design-demo/src/model/tegemoetkoming-schade-covid19.flint.json.old:1]: Yeah commited by accident

**Comment by hidde-jan** [comment]
The newest version of the model (including my fix for horeca) is now included in this PR. I'm pretty sure it's based on the last version Robert posted in slack.

**Pull request merged**